pkg/config/config.go

```diff
@@ -17,24 +17,79 @@ import (
 )
 
 const (
-	maxUInt16 uint16 = 0xFFFF
-	minUInt16 uint16 = 0x0000
+	maxUInt16              uint16 = 0xFFFF
+	minUInt16              uint16 = 0x0000
+	RunInfoFilename               = "bacalhauServe.run"
+	RunInfoFilePermissions        = 0600
 )
 
-func DevstackShouldWriteEnvFile() bool {
-	return DevstackEnvFile() != ""
+var RunInfoFilePath = ""
+
+func DevstackGetShouldPrintInfo() bool {
+	return os.Getenv("DEVSTACK_PRINT_INFO") != ""
+}
+
+func DevstackSetShouldPrintInfo() {
+	os.Setenv("DEVSTACK_PRINT_INFO", "1")
 }
 
 func DevstackEnvFile() string {
 	return os.Getenv("DEVSTACK_ENV_FILE")
 }
 
-func DevstackGetShouldPrintInfo() bool {
-	return os.Getenv("DEVSTACK_PRINT_INFO") != ""
+// WriteRunInfoFile writes the ShellVariables to a file that can be imported. It writes bacalhauServe.run to TempDir().
+func WriteRunInfoFile(ctx context.Context, summaryShellVariablesString string) error {
+	writePath := os.TempDir()
+
+	// TODO: Use the below when we can figure out how to test write permissions in a cross platform way.
+	// See this - https://stackoverflow.com/questions/20026320/how-to-tell-if-folder-exists-and-is-writable //nolint:lll
+	// if _, err := os.Stat("/run"); err == nil {
+	// 	writePath = "/run" // Linux
+	// } else if _, err := os.Stat("/var/run"); err == nil {
+	// 	writePath = "/var/run" // Older Linux
+	// } else if _, err := os.Stat("/private/var/run"); err == nil {
+	// 	writePath = "/private/var/run" // MacOS
+	// } else {
+	// 	// otherwise write to temp dir, which should be available on all systems
+	// 	writePath = os.TempDir()
+	// }
+
+	RunInfoFilePath = filepath.Join(writePath, RunInfoFilename)
```

Comment:
```suggestion
	// TODO: Use the below when we can figure out how to test write permissions in a cross platform way.
	// See this - https://stackoverflow.com/questions/20026320/how-to-tell-if-folder-exists-and-is-writable //nolint:lll
	// if _, err := os.Stat("/run"); err == nil {
	// 	writePath = "/run" // Linux
	// } else if _, err := os.Stat("/var/run"); err == nil {
	// 	writePath = "/var/run" // Older Linux
	// } else if _, err := os.Stat("/private/var/run"); err == nil {
	// 	writePath = "/private/var/run" // MacOS
	// } else {
	// 	// otherwise write to temp dir, which should be available on all systems
	// 	writePath = os.TempDir()
	// }

	RunInfoFilePath = DevstackEnvFile() 
	if RunInfoFilePath == "" {
	    filepath.Join(writePath, RunInfoFilename)
	 }
```

Doing this would mean we can, in dev, override the name of the file being written so that we can get the CLI to auto-load it (https://github.com/bacalhau-project/bacalhau/blob/main/main.go#L26-L30). In prod it would carry on as previously.